.eslintrc.js

```diff
@@ -45,8 +58,12 @@ module.exports = {
     '@intlify/vue-i18n/no-raw-text': 'off',
     'vue/multi-word-component-names': 'off',
     'vue/no-v-model-argument': 'off',
+    'vue/no-reserved-component-names': 'off',
```

Comment:
Why do we need to disable this?

Reply:
This component names would need to be renamed:
"Avatar"
"Badge"
"Card""Filters"
"Header""Loader"
"Modal"
"Confirm"
"Default"
"Help""Panel"
"Platforms"
"Scrollable"
"Tokens"
"Tooltip"
"Truncate"
"Tab"
"Tabs"
"About"
"Address"
"Dashboard"
"Index"
"Invite"
"More"
"Auction"
"Claim"
"Networks"
"Notifications"
"Connect"
"Retip"
"Settings"

I don't want to make this PR even more heavy.